README.md

```diff
@@ -63,7 +63,18 @@ To run an Optic query (either DSL or serialized):
 
 1. Type a valid query in the editor.
 2. Open the command palette (<kbd>Shift</kbd>+<kbd>Cmd</kbd>+<kbd>P</kbd>).
-3. Select one of the `MarkLogic: Submit Optic query - <Response Format>` commands, depending on the desired response format.
+3. Select one of the `MarkLogic: Submit Optic Query - <Response Format>` commands, depending on the desired response format.
+
+Query results will apper in the `MLXPRS: RESULTS` tab in the bottom panel, or open in a new editor tab - depending on the value of the `Marklogic: Results In Editor Tab` setting.
+
+
+### Submit GraphQL Queries
+
+To run a GraphQL query:
```

Comment:
I think making a link to https://docs.marklogic.com/REST/GET/v1/rows/graphql would be helpful here.